### PR TITLE
Show notes on admin/auctions#show

### LINKED ISSUE #1023: Admin view - Auction Show Page - Notes field
This story is to display the notes (if there are any) in a box beneath the status box on the auction show page. See the attached screenshot. This is only visible to admins, and it only shows for auctions that have notes. We may want to name this admin_notes or something a bit more obvious

micropurchase.18f.gov/admin/auctions/[id]

screen shot 2016-08-09 at 10 33 34 am

### PR DESCRIPTION
**Pull request by jessieay**
* Closes #1023

When there are notes:

screen shot 2016-09-07 at 3 01 13 pm

When there are no notes:

screen shot 2016-09-07 at 3 02 26 pm

### CODE REVIEW

File: app/view_models/admin/auction_show_view_model.rb [hunk 1]
```diff
@@ -60,10 +68,6 @@ def title
     auction.title
   end
 
-  def summary
```
Comment (harrisj) [inline app/view_models/admin/auction_show_view_model.rb]: Getting rid of these because they aren't being used?

Comment (jessieay) [inline app/view_models/admin/auction_show_view_model.rb]: yep!

File: app/view_models/admin/auction_show_view_model.rb [hunk 2]
```diff
@@ -82,10 +86,6 @@ def html_summary
     MarkdownRender.new(auction.summary).to_s
   end
 
-  def admin_edit_auction_partial
```
Comment (harrisj) [inline app/view_models/admin/auction_show_view_model.rb]: So we don't have the Edit button on admin auction show?

Comment (jessieay) [inline app/view_models/admin/auction_show_view_model.rb]: we do, I just inlined this.  (` <%= render partial: 'auctions/edit_auction_link', locals: { view_model: @view_model } %>` ) I think this method was leftover from a world where we shared view models between admin and regular auction  show pages.

Comment (harrisj) [inline app/view_models/admin/auction_show_view_model.rb]: Got it. Thanks!

**Comment by harrisj** [comment]
LGTM. Merging

**Pull request merged**